contents/docs/api/post-only-endpoints.mdx

```diff
@@ -258,7 +258,47 @@ Body:
 }
 ```
 
-### Example request & response: /decide v2
+### Example request & response: /decide v3
+`/decide` version 3 introduces support for [feature flag payloads](TODO: update link) and ...
```

Comment:
@EDsCODE @neilkakkar whenever the payload docs stuff is ready ~ and for whatever other stuff we put in decide v3 that I don't remember off the top of my head that's different from decide v2 😳

Reply:
Will update when this goes in https://github.com/PostHog/posthog.com/pull/5682